backend/danswer/connectors/confluence/connector.py

```diff
@@ -286,8 +314,73 @@ def _fetch(start_ind: int, batch_size: int) -> Collection[dict[str, Any]]:
 
                 return view_pages
 
+        def _fetch_child_pages(
+            start_ind: int, batch_size: int
+        ) -> Collection[dict[str, Any]]:
+            get_page_by_id = make_confluence_call_handle_rate_limit(
+                confluence_client.get_page_by_id
+            )
+
+            child_pages: list[dict[str, Any]] = []
+
+            if self.index_origin:
```

Comment:
since this is called on every batch, this will make the root page be included in every batch which is probably not what we want